src/XMakeTasks/Copy.cs

```diff
@@ -408,6 +409,20 @@ CopyFileWithState copyFile
                 _copiedFiles = new TaskItem[0];
                 return true;
             }
+            
+            if (_destinationFiles != null)
+            {
+                var sameDestinationFiles = _destinationFiles.GroupBy(x => x).Where(g => g.Count() > 1).Select(y => y.First());
```

Comment:
I'm currently trying this
```
var sameDestinationFiles = _destinationFiles.GroupBy(x => x.ItemSpec).Where(g => g.Count() > 1);

                foreach(var item in sameDestinationFiles)
                {
                    var log = "SourceFiles ";

                    foreach(var subitem in item)
                    {
                        log += subitem.GetMetadata("OriginalItemSpec");
                    }

                    log += "all have the destination " + item.Key + ".";
                }
```
Problem here is that i don't have access to `OriginalItemSpec` because it is added [later](https://github.com/maddin2016/msbuild/blob/e0d90d241e267e223fc60254cb45713c2fdb85fd/src/XMakeTasks/Copy.cs#L572) 😞